docs/vocascan-server/configuration.md

```diff
@@ -107,6 +107,16 @@ For more information about the logger section see [logging guide](vocascan-serve
 | `max_files`    |         | Limit the number of files created when the size of the log file is exceeded.                                                           |
 | `archive_logs` |         | If true, all log files, but the current one will be packed into a zip archive.                                                         |
 
+## Custom Pages (`pages.*`)
+
+| Name       | Default | Description                                                                                                                                                                                                                                             |
+| ---------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `url`      |         | The route on which your page will be displayed relative to `/`                                                                                                                                                                                          |
+| `fallback` |         | Defines a fallback page, if neither a language is specified per query parameter (`?lang=`), nor the queried language exists. You can either set a static page or a redirect to one of the defined languages (recommended if you use multiple languages) |
+| `type`     |         | Defines whether the server renders a static page or just redirects to another. Options: `file` or `redirect`                                                                                                                                            |
+| `location` |         | Either the path to your html file, or an internet address for the redirect                                                                                                                                                                              |
+| `langs`    |         | Option to define multiple languages for your static file or redirect. Add props, named after the country code (ISO-639-1: Germany -> de), which include a `type` and a `location`                                                                       |
```

Comment:
can you extract type and location into two different rows. Maybe `(fallback|langs).type` or only `lang.type` and then mention in fallback, that the props from langs are valid. And additionally I would suggest to add a link to that ISO norm.